platforms/nuttx/src/px4/common/tasks.cpp

```diff
@@ -88,7 +89,8 @@ int px4_task_spawn_cmd(const char *name, int scheduler, int priority, int stack_
 
 int px4_task_delete(int pid)
 {
-	return task_delete(pid);
```

Comment:
This is still used in `usr_hrt`. If we remove it here we must also change the implementation in `usr_hrt`, and mavlink.